NEWS

```diff
@@ -101,6 +101,12 @@ CHANGES WITH 258 in spe:
           IPv4DuplicateAddressDetectionTimeoutSec=. The default timeout value
           has been changed from 7 seconds to 200 milliseconds.
 
+        * systemd-networkd previously emitted the machine ID as chassis ID
+          through LLDP protocol, but now emits a random (but based on the
```

Comment:
it's not "random" hence. 

maybe say "now emits a deterministic ID, cryptographically derived from the machine ID" or so.